go.sum

```diff
@@ -234,6 +234,8 @@ github.com/pkg/sftp v1.13.1/go.mod h1:3HaPG6Dq1ILlpPZRO0HVMrsydcdLt6HRDccSgb87qR
 github.com/planetscale/planetscale-go v0.51.0/go.mod h1:+rGpW2u7iQZZx4O/nFj4MZe4xIS22CVegEgl1IkTExQ=
 github.com/planetscale/planetscale-go v0.92.0 h1:ifrZCSifooYuM2bG4v+RH34H2MeHQg5m+lu65EtZ9Bw=
 github.com/planetscale/planetscale-go v0.92.0/go.mod h1:fZZ8IJEB1IMrQCt3eo7D4E+hx5zt8mM3z8/aL/p1fiA=
+github.com/planetscale/planetscale-go v0.93.0 h1:cogKCS4pEbHqurZWiPROaEAo16ZEPmRTSdd2IZioa8A=
+github.com/planetscale/planetscale-go v0.93.0/go.mod h1:MR+LZLhZTIzmi2x91nwLQyIedMFhn6sgQDAKs0wAGNQ=
```

Comment:
@mscoutermarsh I think we need a `go mod tidy` here?

Reply:
💯